packages/tools/nodeEditor/src/components/preview/previewManager.ts

```diff
@@ -284,12 +289,33 @@ export class PreviewManager {
         }
     }
 
+    private _hdrSkyBox: Mesh;
+    private _hdrTexture: CubeTexture;
+    private _prepareBackgroundHDR() {
+        if (this._hdrTexture == null) {
+            this._hdrTexture = CubeTexture.CreateFromPrefilteredData("environment.dds", this._scene);

```

Comment:
We do not have any uptodate dds with our current rendering state. It would either be a .hdr or a .env that I would favor a lot.

Reply:
Yes, it's getting harder and harder to author dds files. Lys no longer works with Windows 11 (and later 10 builds) and they seem to have no support for the software anymore. I would prefer to see .hdr and .env as well so that we keep parity with the most recent support for environment files.